active_elements/introduction.rst

```diff
@@ -24,7 +24,7 @@ elements in the A+ frontend requires that the feedback is wrapped in
 a ``<div id="feedback">`` element.
 
 The active element RST directives, ``ae-input`` and ``ae-output``, are documented in the
-`a-plus-rst-tools <https://github.com/Aalto-LeTech/a-plus-rst-tools#6-active-element-input>`_.
+`a-plus-rst-tools <https://git.io/JJ0XS>`_.
```

Comment:
Why a short link?

Reply:
It is more concise?

Reply:
It also obfuscates the destination, thus the reader doesn't klnow where the link will lead to. Short links are only useful when people need to manually write them to somewhere (e.g. when printed on paper).

Reply:
mmmm I thought that having short links might help us to organize the document. But, yes you are right maybe having short links are not a good idea :slightly_smiling_face: I will change it.